pkg/chartverifier/profiles/profile.go

```diff
@@ -88,30 +89,30 @@ func New(config *viper.Viper) *Profile {
 		}
 	}
 
-	var profile *Profile
 	vendorProfiles := profileMap[profileVendorType]
-	defaultProfile := vendorProfiles[0]
-
-	if len(vendorProfiles) > 1 {
-		profile = nil
-		for _, vendorProfile := range vendorProfiles {
-			if len(profileVersion) > 0 {
-				if semver.Compare(semver.MajorMinor(vendorProfile.Version), semver.MajorMinor(profileVersion)) == 0 {
-					profile = vendorProfile
-					break
+	if len(vendorProfiles) > 0 {
+		profileInUse = vendorProfiles[0]
+
+		if len(vendorProfiles) > 1 {
```

Comment:
Instead of nesting like this can we we just check above if len(vendorProfiles) == 1 we set to vendorProfiles[0]. Then this will be just an els if but this time checking for  len(vendorProfiles) > 1. No need to set to nil cause you will default to default profile above this like

Reply:
checking for len being greater than 0 protects against profile.vendorype being set to an invalid value which means there are no vendor profiles in the map.

Reply:
Changed it to use:
``` 
	if vendorProfiles,ok := profileMap[profileVendorType]; ok {
             ......
        } 
```

to be clearer

Reply:
OK looks good